src/Query/QueryBuilder.php

```diff
@@ -17,7 +17,7 @@ class QueryBuilder
     /** @var Filter[] */
     protected $filterCollection;
 
-    /** @var MatchQuery[] */
+    /** @var Query[] */
     protected $matchCollection;
```

Comment:
Maybe the property can be renamed as it doesn't `MatchQuery` anymore ?

Reply:
Once again, as it is protected, it might have been extended and therefore needed to keep from breaking BC